.github/workflows/blas.yml

```
@@ -0,0 +1,64 @@
+name: CI
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+jobs:
+  build:
+    strategy:
```

The current name for this workflow and job collides with our existing CI.

```suggestion
name: BLAS
on:
  push:
    branches: [ 'master' ]
  pull_request:
jobs:
  testing:
    strategy:
```